lib/spreewald/web_steps.rb

```diff
@@ -463,7 +463,18 @@
 #
 # Attention: Doesn't work with Selenium, see https://github.com/jnicklas/capybara#gotchas
 Then /^I should get a download with filename "([^\"]*)"$/ do |filename|
-  expect(page.response_headers['Content-Disposition']).to match /filename="#{Regexp.escape(filename)}"(;|$)/
+  content_disposition = page.response_headers['Content-Disposition']
+  expect(content_disposition).to be_present
+
+  fields = content_disposition.split(/;\s*/)
+
+  # Rails 6+ encodes filenames as defined in https://tools.ietf.org/html/rfc5987.
+  # If available, we prefer the UTF-8 filename.
+  download_filename = [/^filename\*=UTF-8''(.+)/, /^filename="(.+)"/].each do |regexp|
```

Comment:
I think we should further narrow down the regular expression to exclude double quotes. `/^filename="(.+)"/` would match `foo" baz="bar` if the input is `filename="foo" baz="bar"`.

I'd suggest `/^filename="([^"]+)"/`` instead.

Reply:
The first (UTF-8) RegEx seems rather magical too me, but matches the RFC :+1:

Reply:
Thanks for your feedback. Let's go for `filename="(.+?)"`, I really don't like the `"([^"]+)"` pattern. Result will be the same.